### PR TITLE
PowerToys Run: Add new system plugin command to docs

### PR DESCRIPTION
**Pull request by htcfreek**
This PR adds the new command in System plugin of PT Run to the docs page.
The new command will be included in version 0.55.0 and the PR can be merged after the release of the new PT version.

@crutkas can you please review the docs update. Thank you.

### CODE REVIEW

**Comment by PRMerger16** [comment]
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by mattwojo** [comment]
@crutkas to confirm merging

**Comment by crutkas** [comment]
On hold until .55 is released later this month

File: hub/powertoys/run.md [hunk 1]
```diff
@@ -74,15 +74,16 @@ These default activation phrases will force PowerToys Run into only targeted plu
...
+| Action command | Action | Note |
+| :--- | :--- | :--- |
+| `Shutdown` | Shuts down the computer | |
+| `Restart` | Restarts the computer | |
+| `Sign Out` | Signs current user out | |
+| `Lock` | Locks the computer | |
+| `Sleep` | Sleeps the computer | |
+| `Hibernate` | Hibernates the computer | |
+| `Empty Recycle Bin` | Empties the recycle bin | |
+| `UEFI Firmware Settings` | Reboot computer into UEFI Firmware Settings | Only available on systems with UEFI firmware.<br>(Requires administrative permissions.) |
```
Comment (crutkas) [inline hub/powertoys/run.md]: small but br should be <br /> so properly closed.

**Review by crutkas — APPROVED** [review]

**Comment by crutkas** [comment]
Hold until 0.55 release, ETA Jan 31, 2022

File: hub/powertoys/run.md (hunk 1 above)
Comment (htcfreek) [reply hub/powertoys/run.md]: > small but br should be `<br />` so properly closed.

**Comment by PRMerger20** [comment]
@htcfreek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@crutkas Today is 1/31. Do you want this PR merged? Thanks.

**Comment by crutkas** [comment]
this can go live now

**Pull request merged**